misc/userscripts/open_download

```diff
@@ -94,7 +95,7 @@ fi
 line=$(printf "%s\n" "${entries[@]}"  \
        | crop-first-column 55 \
        | column -s $'\t' -t \
-       | $ROFI_CMD "${rofi_default_args[@]}" $ROFI_ARGS) || true
+       | $ROFI_CMD "${rofi_default_args[@]}" "$ROFI_ARGS") || true
```

Comment:
I can confirm what @blyxxyz addresses: `$ROFI_ARGS` is passed to the `open_download` script by the environment which supports only plain strings but no arrays. Hence the ugly hack of splitting `$ROFI_ARGS` again at spaces.